lib/Ix/App.pm

```diff
@@ -183,12 +246,23 @@ sub to_app ($self) {
   return $app;
 }
 
+=method log_access($request, $response, $ctx = undef)
+
```

Comment:
I didn't have anything here because it just wraps the next two methods (which are documented), but of course you wouldn't know that by reading the docs! I'll add a thing.